crates/kitsune_p2p/dht/src/region_set/ltcs.rs

```diff
@@ -107,9 +109,8 @@ impl RegionCoordSetLtcs {
 /// The coordinates for the regions are specified by a few values.
 /// The data to match the coordinates are specified in a 2D vector which must
 /// correspond to the generated coordinates.
-#[derive(serde::Serialize, serde::Deserialize, Derivative)]
+#[derive(Clone, serde::Serialize, serde::Deserialize, Derivative)]
```

Comment:
see above.

maybe this instead?

```suggestion
#[derive(serde::Serialize, serde::Deserialize, Derivative)]
#[cfg_attr(feature = "test_utils", derive(Clone))]
```